---
 kernel/latencytop.c |    3 +++
 1 file changed, 3 insertions(+)

Index: linux-2.6.24-rc7/kernel/latencytop.c
===================================================================
--- linux-2.6.24-rc7.orig/kernel/latencytop.c
+++ linux-2.6.24-rc7/kernel/latencytop.c
@@ -168,6 +168,9 @@ void __sched account_scheduler_latency(s
 	if (inter && usecs > 5000)
 		return;
 
+	if (usecs > 100 && panic_timeout)
+		show_stack(tsk, NULL);
+
 	/* no reason for the latency... bail for now. Later we'll
 	 * store a backtrace item instead
 	 */
